src/Switch/Switch.stories-old.mdx

```diff
@@ -2,7 +2,7 @@ import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
 import { withKnobs, boolean, select } from '@storybook/addon-knobs';
```

Comment:
Should this file be deleted?

Reply:
Was keeping this in as reference for needing to improve the current switch stories